src/state/recordedState.ts

```diff
@@ -56,6 +56,8 @@ export class RecordedState {
       this.commandList.length > 0
     ) {
       // Used for the registers and macros that only record on commandList
+      // The commandList here saves the ex command, which typically do not include <leader>.
+      // Therefore, we do not need the following replace
```

Comment:
I think this would be a bit clearer than referring to "the following replace".

```suggestion
      // The commandList here saves the ex command, so we should not regex replace the
      // leader key by the literal string '<leader>', as would have been done for a normal command.
```